packages/devtools_app/integration_test/test/app_test.dart

```diff
@@ -0,0 +1,78 @@
+// Copyright 2022 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:devtools_app/devtools_app.dart';
+import 'package:devtools_app/src/app.dart';
+import 'package:devtools_app/src/framework/landing_screen.dart';
+import 'package:devtools_app/src/framework/release_notes/release_notes.dart';
+import 'package:devtools_app/src/shared/primitives/simple_items.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:integration_test/integration_test.dart';
+
+import 'test_utils.dart';
+
+void main() {
+  IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  late TestApp testApp;
+
+  setUpAll(() {
+    testApp = TestApp.fromEnvironment();
+    expect(testApp.vmServiceUri, isNotNull);
+  });
+
+  testWidgets('connect to app and switch tabs', (tester) async {
+    await pumpDevTools(tester);
+    expect(find.byType(LandingScreenBody), findsOneWidget);
+    expect(find.text('No client connection'), findsOneWidget);
+
+    // temporary, just committed this to verify that the bots will fail if this
+    // test case fails.
+    expect(true, isFalse);
```

Comment:
Commenting as a reminder to remove :)

Reply:
TODO to myself before landing - make sure the test fails on the CI when the integration test fails.